_includes/schema.html

```diff
@@ -3,23 +3,24 @@
 - schema, array of:
     - key
     - description
+    - requirement (optional)
     - type (optional)
     - properties (optional)
 {% endcomment %}
 <ul>
   {% for item in include.schema %}
-    <li>
-<div markdown="1">
-**{{ item.key }}**
-{%- if item.type -%}
-  ({{ item.type }})
-{%- endif -%}
-<br />
-{{ item.description }}
-</div>
-      {% if item.properties %}
-        {% include schema.html schema=item.properties %}
-      {% endif %}
+    <li markdown="1">
+<!-- -->**{{ item.key }}**
```

Comment:
this is my fake indentation that lets maintain markdown formatting... I think I may just give up and not markdown it... changes incoming

Reply:
nevermind... this is the best way to get jekyll to not stick a giant `<p>` tag at the start of the description, which ends up adding way too much visual space